components/DeviceList/deviceList.js

```diff
@@ -5,7 +5,7 @@ import Link from "next/link";
 import styled from "styled-components";
 
 //Styling
-const StyledWrapper = styled.div`
+export const StyledWrapper = styled.div`
```

Comment:
```suggestion
const StyledWrapper = styled.div`
```
I don't think you need to export `StyledWrapper`!?

Reply:
yes you`re right, i was just trying to test something...